src/app/elements/MaterialViewElement.js

```diff
@@ -83,7 +83,9 @@ export default class MaterialViewElement extends BaseElement {
     display: flex;
   }
 </style>
-<title-bar title="Material View"></title-bar>
+<title-bar title="Material View">
+  <devtools-icon-button sheet-variant="large" icon-style="--spritesheet-position:-84px 48px; width: 28px; height: 24px;">
```

Comment:
I think I'll move the icon span into its own class and move the ui-icon.js styles to there as well.  Maybe it could look something like `<devtools-icon glyph="refresh"></devtools-icon>` ?